### PR TITLE
MGMT-1667: added minikube profile flag

### PR DESCRIPTION
**Pull request by RazRegev**
In order to allow multiple deployments on same host, a profile flag will
determent on which minikube profile to create the new deployment. In this
commit this flag was added to all relevant minikube commands.

### CODE REVIEW

File: tools/deployment_options.py [hunk 1]
```diff
@@ -12,7 +12,24 @@ def load_deployment_options(parser=None):
...
+    )
+    parser.add_argument(
+        '--profile',
+        help='Profile for all deployment images',
+        type=str,
+        default='minikube'
+    )
+    parser.add_argument(
+        '--target',
+        help='Target deployment (minikube/oc/oc-ingress)',
```
Comment (YuviGold) [inline tools/deployment_options.py:29]:
```python
choices=['minikube', 'oc', 'oc-ingress']
```

File: tools/utils.py [hunk 2]
```diff
@@ -11,13 +11,31 @@
...
 PODMAN = "podman"
 
+
 def check_output(cmd):
     return subprocess.check_output(cmd, shell=True).decode("utf-8")
 
 
-def get_service_host(service, target=None, domain="", namespace='assisted-installer'):
+def set_profile(target, profile):
+    if target != 'minikube':
```
Comment (YuviGold) [inline tools/utils.py:20]: create constant variable

File: tools/deployment_options.py [hunk 3]
```diff
@@ -12,7 +12,24 @@ def load_deployment_options(parser=None):
     if not parser:
         parser = argparse.ArgumentParser()
 
-    parser.add_argument('--namespace', help='Namespace for all deployment images', type=str, default='assisted-installer')
```
Comment (YuviGold) [inline tools/deployment_options.py:15]: I think you can add the `--domain` flag also in here while it is broadly used

File: tools/utils.py (hunk 2 above)
Comment (RazRegev) [reply tools/utils.py:20]: why? don't think there is a need to use a constant

**Pull request merged**